### ISSUE TITLE
phpfastcache speed

### ISSUE THREAD

**Issue by belliash** [PHPSocialNetwork/phpfastcache#237]
Could you prove somehow it is really faster? Is your timing based on any benchmark, as I couldn't find any comparison on website, or is that just your saying?

**Comment by khoaofgod** [comment]
we don't have time for it right now, still have many other functions are on developing. However if u want, u can do a loop 1 to 50,000 , and use phpFastCache with another normal file cache, use microtime and try call / get / write and see

**Comment by belliash** [comment]
This is not a benchmark. If a client connects to your website, some data are fetched from database and then saved in cache. In *MOST* if not all situations, you use cached variable once per request. I cant image you would need to access metadata of article several times. Even so, once you grab them from cache and store in variable/registry/etc... they can be used all the time during script execution.

Now, when another client connects again, all variables are empty at the beginning of script execution. Again, there is a need to access cache.

To sum up, since the values saved in variables are available only per request, I doubt this kind of benchmark is valuable and your cache implementation really faster. Thus I think this is just a bullshit.
Am I wrong? Prove it.

**Comment by belliash** [comment]
Or as I see, it is much easier to close the ticket instead . . .
